test/unit/orm-adapters-memory-test.js

```diff
@@ -0,0 +1,14 @@
+import test from 'ava';
+import { MemoryAdapter } from '../../lib/data/orm-adapters/memory';
```

Comment:
This isn't working for me at the moment, can I get some guidance on what it should be instead?